apps/builder/app/shared/canvas-api.ts

```diff
@@ -0,0 +1,71 @@
+import { setInert, resetInert } from "../canvas/shared/inert";
+
+const _canvasApi = {
```

Comment:
since this is on canvas where we will have 3rd party code running, I would use some namespace that looks like __webstudioSomething